jaxrs/src/main/java/org/killbill/billing/jaxrs/json/SubscriptionJson.java

```diff
@@ -135,7 +135,7 @@ public EventSubscriptionJson(final SubscriptionEvent subscriptionEvent, @Nullabl
             final PlanPhase phase = subscriptionEvent.getNextPhase() != null ? subscriptionEvent.getNextPhase() : subscriptionEvent.getPrevPhase();
             this.eventId = subscriptionEvent.getId();
             this.billingPeriod = billingPeriod;
-            this.effectiveDate = subscriptionEvent.getEffectiveDate();
+            this.effectiveDate = subscriptionEvent.getEffectiveDate() != null ? subscriptionEvent.getEffectiveDate().toLocalDate() : null; //TODO_1375 - Added this to keep the jaxrs layer the same, will be updated in step 2. Not sure how to access internalCallContext here so I've used toLocalDate
             this.plan = plan != null ? plan.getName() : null;
```

Comment:
Again, need your input here. I'm converting `DateTime` to `LocalDate` to keep the jaxrs layer the same like we discussed. However, I was not sure how to access `internalCallContext` in the `SubscriptionJson` class, so I've used the  `toLocalDate()` method.

Reply:
Ok, keep the TODO